src/ActionModal/components/ActionModal.vue

```diff
@@ -672,11 +673,13 @@ export default {
         this.selectedSignMethod
       )
       this.$emit(`txIncluded`)
+      this.$apollo.queries.overview.refetch()
```

Comment:
@colw for your info - without this, second tx's fail because the context isn't recreated and the sequence number stays the same.